content/applications/inventory_and_mrp/inventory/routes/concepts/stock_warehouses.rst

```diff
@@ -1,47 +1,99 @@
+
+Create a new warehouse
+----------------------
 
-.. image:: stock_warehouses/ware1.png
+To create a new warehouse, go to :menuselection:`Inventory --> Configuration --> Warehouses`, and
+click :guilabel:`New`. From here, the warehouse name can be changed, as well as other details about
+how to resupply the warehouse. Routes can also be set by clicking on the :guilabel:`Routes` smart
+button. Once the warehouse is configured, virtual locations can be created.
+
+.. image:: stock_warehouses/stock-warehouses-create-warehouse.png
    :align: center
+   :alt: The edit screen for creating a new warehouse.
 
-.. note::
-   The *Storage Location* feature will be automatically enabled. Good news, because you will need it
-   later in the process.
+Create a virtual parent location
+--------------------------------
+
+.. important::
+   In order to take stock from multiple warehouses to fulfill a sales order, there need to be at
+   least **two** warehouses acting as children locations of the virtual parent location warehouse.
 
-Now, you have to make sure that the main stock locations of warehouse A and warehouse B are children
-locations of the main stock location of warehouse A + B. Go to the locations menu, and edit the main
-location of your two warehouses. Then, change their parent location to main location of warehouse
-A+B.
+To create and edit locations, go to :menuselection:`Inventory --> Configuration --> Locations`. All
+locations are listed here,including the stock location of the virtual warehouse that was created.
+Click into the stock location for the virtual warehouse created previously, and under the
+**Additional Information** tab, change the :guilabel:`Location Type` from *Internal Location* to
+*View*. This identifies this location as a virtual one used to create a hierarchical structure for
+a warehouse, aggregating its child locations. Products can *not* be stored in this
+:guilabel:`Location Type`.
 
-.. image:: stock_warehouses/ware2.png
+.. image:: stock_warehouses/stock-warehouses-location-types.png
    :align: center
+   :alt: Warehouse location types in location creation screen.
 
-Sell a product from the virtual warehouse
-=========================================
+Configure physical warehouse locations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Let’s say you have two products, one stored in warehouse A and one stored in warehouse B. Now, you
-can create a new quotation for one of each product. Go to other information and choose Warehouse A+B
-in the shipping information.
+Navigate back to the :guilabel:`Locations` overview via the breadcrumbs, then click into the first
+physical warehouse stock location. Under :guilabel:`Parent Location`, select the virtual warehouse
+from the drop-down menu. Navigate back to the :guilabel:`Locations` overview and repeat this step
+for the second physical warehouse stock location. Both locations are now child locations of the
+virtual warehouse parent location.
```

Comment:
Navigate back to the :guilabel:`Locations` overview (via the breadcrumbs), then click into the first
physical warehouse stock location. 

Under :guilabel:`Parent Location`, select the virtual warehouse
from the drop-down menu. Then, navigate back to the :guilabel:`Locations` overview, and repeat this step
for the second physical warehouse stock location. 

Both locations are now "Child" locations of the virtual warehouse "Parent" location.